--- Cartopy-0.18.0.orig/lib/cartopy/mpl/geoaxes.py	2020-05-03 23:58:12.000000000 -0300
+++ Cartopy-0.18.0/lib/cartopy/mpl/geoaxes.py	2021-03-04 18:26:13.678823759 -0300
@@ -469,11 +469,12 @@
         #       caching the resulting image;
         #       buffering the result by 10%...;
         if not self._done_img_factory:
-            for factory, args, kwargs in self.img_factories:
+            for factory, factory_args, factory_kwargs in self.img_factories:
                 img, extent, origin = factory.image_for_domain(
-                    self._get_extent_geom(factory.crs), args[0])
+                    self._get_extent_geom(factory.crs), factory_args[0])
                 self.imshow(img, extent=extent, origin=origin,
-                            transform=factory.crs, *args[1:], **kwargs)
+                            transform=factory.crs, *factory_args[1:],
+                            **factory_kwargs)
         self._done_img_factory = True
 
         return matplotlib.axes.Axes.draw(self, renderer=renderer, **kwargs)
