--- Cartopy-0.18.0b1.orig/setup.py	2020-02-10 21:40:13.000000000 -0300
+++ Cartopy-0.18.0b1/setup.py	2020-02-11 10:37:23.063913238 -0300
@@ -241,7 +241,7 @@
     This function gets the PROJ libraries to cythonize with
     """
     proj_libraries = ["proj"]
-    if os.name == "nt" and proj_version >= (6, 0, 0):
+    if os.name == "nt" and proj_version >= (6, 0, 0) and proj_version < (6, 3, 0):
         proj_libraries = [
             "proj_{}_{}".format(proj_version[0], proj_version[1])
         ]
